ticdc/confluent-integration-using-ticdc.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Quick Start Guide on Integrating TiDB with Confluent Platform
```

Comment:
@kissmydb Please help determine where this document should be placed in TOC. Thanks~